lib/paperclip.rb

```diff
@@ -193,6 +193,7 @@ def has_attached_file(name, options = {})
 require 'paperclip/io_adapters/empty_string_adapter'
 require 'paperclip/io_adapters/identity_adapter'
 require 'paperclip/io_adapters/file_adapter'
+require 'paperclip/io_adapters/file_path_adapter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.